github/config.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/github/hub/Godeps/_workspace/src/github.com/howeyc/gopass"
 	"github.com/github/hub/ui"
 	"github.com/github/hub/utils"
+	"github.com/github/hub/Godeps/_workspace/src/github.com/cep21/xdgbasedir"
```

Comment:
My general rule for software is not to pull in dependencies when you can simply inline something. This is the entirety of the logic that we're obtaining by pulling in an external library:

```go
// DataHomeDirectory is a single base directory relative to which user-specific data files should be written
func DataHomeDirectory() (string, error) {
	return getInEnvOrJoinWithHome("XDG_DATA_HOME", defaultDataHomeDirectory)
}
```

we can do the same ourselves. Also, this library uses `user.Current()`, which we want to avoid due to cross-compilation issues: https://github.com/github/hub/pull/1082